entities/person.rb

```diff
@@ -24,6 +25,10 @@ def correct_name
     @name
   end
 
+  def create_rental(_rental)
+    Rental.new(date, self, person)
+  end
+
```

Comment:
- Let's update the method as we only need the date and book that would rented out by the person below
- Change the person to book as the person is renting a book
``` 
  def create_rental(book, date)
    Rental.new(date, self, book)
  end
```

Reply:
Thank you very much for your suggestions !